pkg/storegateway/series_chunks_test.go

```diff
@@ -848,11 +929,13 @@ func (f *chunkReaderMock) load(result []seriesEntry, chunksPool *pool.SafeSlabPo
 		return f.loadErr
 	}
 	for chunkRef, indices := range f.toLoad {
-		// Take bytes from the pool, so we can assert on number of allocations and that frees are happening
-		chunkData := f.chunks[chunkRef].Raw.Data
-		copiedChunkData := chunksPool.Get(len(chunkData))
-		copy(copiedChunkData, chunkData)
-		result[indices.seriesEntry].chks[indices.chunk].Raw = &storepb.Chunk{Data: copiedChunkData}
+		if f.chunks[chunkRef].Raw != nil {
```

Comment:
Can you please add a comment to explain why we wouldn't want to load chunks where the raw chunk is nil?

Reply:
Yep!